Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -476,6 +498,102 @@ public virtual CursorStateEnum CheckCursorState()
             return CursorStateEnum.Contextual;
         }
 
+        /// <summary>
+        /// Virtual function for checking cursor context changes.
+        /// </summary>
+        public virtual CursorContextEnum CheckCursorContext()
+        {
+            // Temp implementation
+            if (CursorContext != CursorContextEnum.Contextual)
+            {
+                if ((CurrentCursorActions & CursorAction.Move) != 0)
+                {
+                    return CursorContextEnum.MoveCross;
+                }
+                else if ((CurrentCursorActions & CursorAction.Scale) != 0)
+                {
+                    if (handleRigTransform != null)
+                    {
+                        Vector3 adjustedHandlePos = handleRigTransform.InverseTransformVector(Position - handleRigTransform.position);
```

Comment:
It looks like you're trying to figure out what type of resize handle to use here, but I'm not fully following how because I don't understand what handleRigTransform is. Could you explain in a comment what you are doing here?

Reply:
Hopefully the new logic makes a bit more sense. I can still add a comment if you feel like it's necessary.

Reply:
`adjustedHandlePos` is still a bad name. That needs changing.